src/components/Recipe.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { RiTimerLine } from 'react-icons/ri';
+import { BsPeopleFill } from 'react-icons/bs';
+import { AiFillStar } from 'react-icons/ai';
+import { getRecipe } from '../Redux/recipes/recipes';
+import Loader from './Loader';
+import '../styles/Recipe.css';
+
+const Recipe = () => {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const recipesData = useSelector((state) => state.recipes);
+  const { loader, recipe } = recipesData;
+
+  useEffect(() => {
+    dispatch(getRecipe(id));
+  }, []);
+
+  return (
+    <div>
+      { loader && <Loader /> }
+      { !loader && (
+        <div className="recipe-details">
+          <div className="recipe-banner">
+            { recipe.original_video_url && (
+            <video src={recipe.original_video_url} autoPlay="true" controls="controls" />
+            )}
+            { !recipe.video_url && (
+            <img src={recipe.thumbnail_url} alt={recipe.name} />
+            )}
+          </div>
+          <div className="recipe-infos">
+            <h1>{recipe.name}</h1>
+            <div className="metrics">
+              <span>
+                {' '}
+                <RiTimerLine />
+                {' '}
+                {recipe.prep_time_minutes}
+              </span>
```

Comment:
You did great work here by viewing recipes by category. Still, it would be nice if you placed a back button on the header or somewhere in the recipe component in order to take the user back to either the category page to view another recipe of the same category or to go back to the home page. Currently, you only have the home icon in the header that takes the user directly to the homepage.

- [ ] Imaging you selected Trending category, and then pick on a recipe to view its details, and again you would want to get back to the same category that's showing 8 foods(not the 4 foods that are shown on the home page) and you want to view a different food. That's not possible at the moment. Kindly work on that 👍